### PR TITLE
TF-13302: Adds Runs to Organization Settings

### PR DESCRIPTION
**Pull request by carolinaborim**
### What

`/terraform/cloud-docs/users-teams-organizations/organizations` now will have Runs menu, where the new functionality is explained

### Why

Since we recently added a new feature for Organization Settings, we are updating the docs to reflect this new state.

### Screenshots

image

This documentation reflects this functionality:
image

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [x] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. HCP Terraform ).
- [x] Description links to related pull requests or issues, if any.

#### Content
- [n/a] Redirects have been added to `website/redirects.js` for moved, renamed, or deleted pages.
- [n/a] API documentation and the API Changelog have been updated. 
- [n/a] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [n/a] Pages with related content are updated and link to this content when appropriate.
- [n/a] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [n/a] New pages have metadata (page name and description) at the top.
- [n/a] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [n/a] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [x] UI elements (button names, page names, etc.) are bolded.
- [ ] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

File: website/docs/cloud-docs/users-teams-organizations/organizations/index.mdx
Comment (trujillo-adam) [inline website/docs/cloud-docs/users-teams-organizations/organizations/index.mdx:158]:
```suggestion
From the workspace home page, click **Settings** in the sidebar, then **Runs**  to view existing runs in all workspaces. You can perform the following actions:

- Click **Needs Attention** to filter runs that are ?
- Click **Running** to filter for runs that are in progress. 
- Click **On Hold** to filter runs that have paused. 
- Click on a column header to display runs in alphanumeric order according to the most recent change or workspace name.  

Refer to [Run States and Stages](/terraform/cloud-docs/run/states) for additional information about understanding run states.
```
I think we should be a little more concrete in describing how to get to this page and how to operate it. It's pretty intuitive, but framing it this way enables us to provide a little more information about the different states. Speaking of which, what does "Needs Attention" mean? Can we provide at least one example of why a run might need attention? I did not see it listed on the Run States and Stages page, but maybe I'm linking to the wrong page? We should also provide at least one possible reason why a run is in an On Hold state.

**Review by trujillo-adam — CHANGES_REQUESTED** [review]
Thanks for adding this. I reworked the content a bit and posted a few questions in the review comment. Let me know if anything is unclear or if you disagree with anything.

File: website/docs/cloud-docs/users-teams-organizations/organizations/index.mdx
Comment (carolinaborim) [reply website/docs/cloud-docs/users-teams-organizations/organizations/index.mdx:158]: Thanks for the quick review @trujillo-adam 

I understand your questions and concerns, but I have a couple of considerations:

1. This piece of documentation is already part (as it is just a new h4) of Organizations Settings, that is why explaining on this specific docs how to get to the page, does not make much sense here.

Regarding `needs attention`, you can read about it in different places, but here is
[a very good example]:(https://developer.hashicorp.com/terraform/cloud-docs/api-docs/notification-configurations)
`Needs Attention - "run:needs_attention" -
A plan has changes and Terraform requires user input to continue. This input may include approving the plan or a [policy override](https://developer.hashicorp.com/terraform/cloud-docs/run/states#the-policy-check-stage).`

I agree with your suggestions, but I am just a little bit concern we differ a lot from the current documentation for that page. I just want to make it smooth and more fluid as possible.

Comment (trujillo-adam) [reply website/docs/cloud-docs/users-teams-organizations/organizations/index.mdx:158]: Thank, I updated the suggestion to provide a description for "Needs Attention". I missed that there is a notifications reference in the API section of the docs. I left the link to the Run States and Stages page, though, because I did not see "On Hold" or "Running" on the notifications API reference. 

As far as providing instructions on how to get to the Runs page, I don't think we can assume that the user started at the top of the page and is working their way down and following along in the UI. Assuming users have found the correct page, they are probably using control-f to find specific references to the "run page" to learn more about it. It's fine if this section is a little different from the others. At some point, we are going to reorganize and refresh all of the content.

Comment (carolinaborim) [reply website/docs/cloud-docs/users-teams-organizations/organizations/index.mdx:158]: @trujillo-adam just two considerations on your latest suggestion:

"From the workspace home page" if we are using this here, we need to make sure we use **workspaceS** in plural, since we are actually accessing Organization Settings and not a workspace settings. This can be a bit confusing, so we need to make sure to use Workspaces.

"to view existing runs in all workspaces" - we need to specifically use _current runs_, instead of existing runs. The difference is that we are not showing existing runs, we are only showing the Current run for each of the workspaces, and these are  different concepts.

Comment (trujillo-adam) [reply website/docs/cloud-docs/users-teams-organizations/organizations/index.mdx:158]: Thank you for clarifying! I updated the suggestion accordingly.

**Comment by carolinaborim** [comment]
@trujillo-adam thanks for the suggestions, they are commited... the PR is ready for re-review/approval.

File: website/docs/cloud-docs/users-teams-organizations/organizations/index.mdx
Comment (trujillo-adam) [inline website/docs/cloud-docs/users-teams-organizations/organizations/index.mdx:156]:
```suggestion
From the Workspaces page, click **Settings** in the sidebar, then **Runs** to view current runs in all workspaces. You can perform the following actions:
```

Comment (trujillo-adam) [reply website/docs/cloud-docs/users-teams-organizations/organizations/index.mdx:156]: Sorry, I introduced a typo

**Review by trujillo-adam — APPROVED** [review]
I made introduced a typo - approving so you can merge when ready

**Pull request merged**
